specification/cognitiveservices/Language.AnalyzeText/models/healthcare.tsp

```diff
@@ -0,0 +1,494 @@
+  @doc("Specifies the method used to interpret string offsets.")
+  stringIndexType?: StringIndexType = StringIndexType.TextElements_v8;
+
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type"
+  @added(Versions.v2023_11_15_preview)
+  @doc("The FHIR Spec version that the result will use to format the fhirBundle. For additional information see https://www.hl7.org/fhir/overview.html.")
+  fhirVersion?: fhirVersion;
+
+  @added(Versions.v2023_11_15_preview)
+  @doc("Document type that can be provided as input for Fhir Documents. Expect to have fhirVersion provided when used. Behavior of using None enum is the same as not using the documentType parameter.")
+  documentType?: healthcareDocumentType;
+}
+
+@doc("The FHIR Spec version.")
+@added(Versions.v2023_11_15_preview)
+union fhirVersion {
+  string,
+
+  @doc("Version 4.0.1")
+  `4.0.1`: "4.0.1",
+}
+
+@doc("Document type.")
+@added(Versions.v2023_11_15_preview)
+union healthcareDocumentType {
+  string,
+
+  @doc("None document type")
+  None: "None",
+
+  @doc("Clinical trial document type")
+  ClinicalTrial: "ClinicalTrial",
+
+  @doc("Discharge summary document type")
+  DischargeSummary: "DischargeSummary",
+
+  @doc("Progress note document type")
+  ProgressNote: "ProgressNote",
+
+  @doc("History and physical document type")
+  HistoryAndPhysical: "HistoryAndPhysical",
+
+  @doc("Consult document type")
+  Consult: "Consult",
+
+  @doc("Imaging document type")
+  Imaging: "Imaging",
+
+  @doc("Pathology document type")
+  Pathology: "Pathology",
+
+  @doc("Procedure note document type")
+  ProcedureNote: "ProcedureNote",
+}
+
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+@doc("The long running task to be performed by the service on the Healthcare input documents.")
+model HealthcareLROTask extends AnalyzeTextLROTask {
+  @doc("Healthcare kind of the long running task.")
+  kind: AnalyzeTextLROTaskKind.Healthcare;
+
+  @doc("Parameters for the Healthcare task.")
+  parameters?: HealthcareTaskParameters;
+}
+
+@doc("Result object for the processed Healthcare task.")
+model HealthcareResult is PreBuiltResult {
+  @typeChangedFrom(
+    Versions.v2023_11_15_preview,
+    Array<HealthcareEntitiesDocumentResult>
+  )
+  @doc("List of result objects for the processed Healthcare documents.")
+  documents: Array<HealthcareEntitiesDocumentResultWithDocumentDetectedLanguage>;
+}
+
+@doc("Result object for the processed Healthcare document.")
+model HealthcareEntitiesDocumentResult is DocumentResult {
+  @doc("Healthcare entities.")
+  entities: Array<HealthcareEntity>;
+
+  @doc("Healthcare entity relations.")
+  relations: Array<HealthcareRelation>;
+
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type"
+  @added(Versions.v2023_11_15_preview)
+  @doc("JSON bundle containing a FHIR compatible object for consumption in other Healthcare tools. For additional information see https://www.hl7.org/fhir/overview.html.")
+  fhirBundle?: Record<unknown>;
```

Comment:
@johanste we have 2 instances where we have this open-ended response. - LUIS result and fhirBundle indicated here. @Dezzley can provide details on whether we can have this response structure declared.

Reply:
For arbitrary FHIR, let's create a 

`model FhirBundle is Record<unknown> {};`

Reply:
@bidisha-c any issues with implementing the suggestion above?